OWNERS_ALIASES

```diff
@@ -0,0 +1,28 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+aliases:
+  slack-infra-approvers:
+    - coderanger
+    - cpanato
+    - jberkus
+    - jeefy
+    - mrbobbytables
+
+  slack-infra-reviewers:
+    - coderanger
+    - cpanato
+    - jberkus
+    - jeefy
+    - mrbobbytables
+
+  sig-contributor-experience-leads:
+    - MadhavJivrajani
+    - Priyankasaggu11929
+    - kaslin
+    - palnabarun
+
+  sig-k8s-infra-leads:
```

Comment:
IMHO: I pinged the current set of leads to invite them to help because we need people from somewhere (and same for contribex), but I don't know that this is strictly related to those roles long term if we gain healthy subproject owners.

Reply:
Did you want to hold this PR then, or approve it?

Reply:
My personal suggestion is that we inline the approvers in OWNERS as just the owners of this repo, but I don't consider it a hard blocker. Nobody currently on the PR can actually approve it.

Reply:
did an update, PTAL